plugins/janus_videoroom.c

```diff
@@ -1431,6 +1300,7 @@ typedef struct janus_videoroom_session {
 	janus_mutex mutex;
 	janus_refcount ref;
 } janus_videoroom_session;
+// janus_plugin_session -> janus_videoroom_session
```

Comment:
I guess this comment here is a leftover? Please remove this. As a side note, per our code style comments should always be in the form of `/* */` and not `//`.